vite.config.js

```diff
@@ -42,6 +42,10 @@ const configAvivator = defineConfig({
   base: './',
   root: 'avivator',
   publicDir: 'avivator/public',
+  define: {
+    'process.env': JSON.stringify(process.env),
+    'global': 'gobalThis',
```

Comment:
I checked-out the branch, this should be:

```suggestion
    'process.env.NODE_DEBUG': false, 
    'global.Symbol': 'globalThis.Symbol',
```

Reply:
Ugh, @ilan this is messing up Vite now (sorry). Pretty annoying that we need to inject custom globals..

I think `'global.Symbol': 'Symbol'` is preferable in this case. IMO it's better to be more specific about what we 'define', so that we can help end-users better